versions/v2.13/modules/en/pages/api/workflows/users.adoc

```diff
@@ -0,0 +1,197 @@
+= Users
+ifeval::["{build-type}" != "srfa"]
+:page-languages: [en, zh]
+endif::[]
+:revdate: 2025-12-10
+:page-revdate: {revdate}
+
+== User Resource
+
+The `User` resource (users.management.cattle.io) represents a user account in Rancher.
+
+To get a description of the fields and structure of the `User` resource, run:
+
+[source,shell]
+====
+kubectl explain users.management.cattle.io
+====
+
+== Creating a User
+
+Creating a local user is a two-step process: you must create the `User` resource, then provide a password via a Kubernetes `Secret`.
+
+Only a user with sufficient permissions can create a `User` resource.
+
+[source,shell]
+====
+kubectl create -f -<<EOF
+apiVersion: management.cattle.io/v3
+kind: User
+metadata:
+ name: testuser     
+displayName: "Test User"             
+username: "testuser"
+EOF
+====
+
+The user's password must be provided in a `Secret` object within the `cattle-local-user-passwords` namespace. The Rancher webhook will automatically hash the password and update the `Secret`.
+
+[IMPORTANT]
+====
+Important: The `Secret` must have the same name as the metadata.name (and username) of the `User` resource. 
+====
+
+[source,shell]
+====
+kubectl create -f -<<EOF
+apiVersion: v1
+kind: Secret
+metadata:
+ name: testuser
+ namespace: cattle-local-user-passwords
+type: Opaque
+stringData:
+ password: Pass1234567!
+EOF
+====
+
+After the plaintext password is submitted, the Rancher-Webhook automatically hashes it, replacing the content of the `Secret`, ensuring that the plaintext password is never stored:
+
+[source,yaml]
+====
+apiVersion: v1
+data:
+  password: 1c1Y4CdjlehGWFz26F414x2qoj4gch5L5OXsx35MAa8=
+  salt: m8Co+CfMDo5XwVl0FqYzGcRIOTgRrwFSqW8yurh5DcE=
+kind: Secret
+metadata:
+  annotations:
+    cattle.io/password-hash: pbkdf2sha3512
+  name: testuser
+  namespace: cattle-local-user-passwords
+  ownerReferences:
+  - apiVersion: management.cattle.io/v3
+    kind: User
+    name: testuser
+    uid: 663ffb4f-8178-46c8-85a3-337f4d5cbc2e
+  uid: bade9f0a-b06f-4a77-9a39-4284dc2349c5
+type: Opaque
+====
+
+== Updating User's Password
+To change a user's password, use the `PasswordChangeRequest` resource, which handles secure password updates.
+
+[source,yaml]
```

Comment:
```suggestion
[source,shell]
```

This is present in the original PR, but it shouldn't be YAML since it's running `kubectl`.